auth/src/main/AndroidManifest.xml

```diff
@@ -15,6 +16,10 @@
             android:name="io.fabric.ApiKey"
             android:value="@string/twitter_consumer_secret"/>
 
+        <meta-data
+            android:name="com.facebook.sdk.ApplicationId"
+            android:value="@string/facebook_application_id"/>
```

Comment:
This fixes the [Chrome Custom Tabs issue](https://youtu.be/pcovdrZhszw) where Facebook would use CCT completely randomly.

See step 8 in the official documentation: https://developers.facebook.com/docs/facebook-login/android